node/full_node_integration_test.go

```diff
@@ -535,25 +563,27 @@ func createNode(ctx context.Context, n int, isMalicious bool, aggregator bool, d
 	}
 
 	genesisValidators, signingKey := getGenesisValidatorSetWithSigner(1)
-	node, err := newFullNode(
+	genesis := &tmtypes.GenesisDoc{ChainID: "test", Validators: genesisValidators}
+	if isLight {
+		// TODO: need to investigate why this needs to be done for light nodes
+		genesis.InitialHeight = 1
+	}
+	node, err := NewNode(
 		ctx,
 		config.NodeConfig{
 			P2P:                p2pConfig,
 			DALayer:            "mock",
 			Aggregator:         aggregator,
 			BlockManagerConfig: bmConfig,
+			Light:              isLight,
 		},
 		keys[n],
 		signingKey,
 		proxy.NewLocalClientCreator(app),
-		&tmtypes.GenesisDoc{ChainID: "test", Validators: genesisValidators},
```

Comment:
don't need to add additional logic for only `isLight`, but you can set `InitialHeight = 1` for all types of nodes as everything starts at height 1.

Reply:
done